webpack.config.dev.js

```diff
@@ -0,0 +1,37 @@
+var path = require('path');
+var webpack = require('webpack');
+// const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;

```

Comment:
Avoid to leave code that is commented. You can create a conditional execution of this code by env variables if necessary

Reply:
Okay.